lib/puppet/interface/action_builder.rb

```diff
@@ -144,6 +150,9 @@ def initialize(face, name, &block)
     @face   = face
     @action = Puppet::Interface::Action.new(face, name)
     instance_eval(&block)
-    @action.when_invoked or raise ArgumentError, "actions need to know what to do when_invoked; please add the block"
+    unless @action.when_invoked
+      #TRANSLATORS 'when_invoked' is a method name and should not be translated and 'block' is a Ruby code block
```

Comment:
Is the comment here just adding context to help translate `block`, or should `block` not be translated?

Reply:
Also, I don't like this error message, I think it doesn't lend itself well to translation. But that's probably neither here nor there. Definitely not something I think we should fix here.

Reply:
I'm inclined to leave this as is.

Reply:
👍